generator.yaml

```diff
@@ -161,3 +161,17 @@ resources:
       Tags:
         compare:
           is_ignored: true
+  OpenIDConnectProvider:
```

Comment:
I think we need to mention the `customUpdateFunction` here. e.g https://github.com/aws-controllers-k8s/ecr-controller/blob/main/generator.yaml#L55-L56

Reply:
Done